python/sglang/srt/constrained/__init__.py

```diff
@@ -52,15 +52,14 @@ def build_regex_from_object(
 
 
 try:
-    from xgrammar import (
-        GrammarMatcher,
-        GrammarMatcherInitContext,
-        GrammarMatcherInitContextCache,
-    )
+    from xgrammar import CachedGrammarCompiler as GrammarMatcherInitContextCache
+    from xgrammar import CompiledGrammar as GrammarMatcherInitContext
+    from xgrammar import GrammarMatcher
```

Comment:
The "as" seems redundant here. We can rename variables in all files.
```
from xgrammar import CachedGrammarCompiler
```

Reply:
Sure. Fixed in the latest commit!